todo_app/templates/index.html

```diff
@@ -18,15 +18,15 @@ <h2>Things to do</h2>
               <td scope="col">Move to done?</td>
             </tr>
         </thead>
-        {{ items.name }}
-        {% for card in items %}
+        {{ view_model.cards.name }}
```

Comment:
Is this line actually printing anything? I don't think `.name` will do anything on the cards list